curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e63bb1e32d23b6adbe44.md

```diff
@@ -271,8 +271,7 @@ function calculateCalories(e) {
 
   const consumedCalories = breakfastCalories + lunchCalories + dinnerCalories + snacksCalories;
   const remainingCalories = budgetCalories - consumedCalories + exerciseCalories;
-  const surplusOrDeficit = remainingCalories >= 0 ? 'Surplus' : 'Deficit';
-  output.innerHTML = ``;
+	const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';  output.innerHTML = ``;
```

Comment:
```suggestion
  const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';
  output.innerHTML = ``;
```

Reply:
the indentations have now been resolved in the latest push